### ISSUE TITLE
Azure not yet supported?

### ISSUE THREAD

**Issue by jlk** [tenable/KaiMonkey#7]
I see mention of azure in the pre-reqs, but no sign of azure tf code...

**Comment by luiz1919** [comment]
Yes I would like to learn this stuff for azure as well, any timelines guys? much appreciated
